code/modules/reagents/chemistry/reagents.dm

```diff
@@ -229,6 +229,14 @@ Primarily used in reagents/reaction_agents
 
 /// Called when an overdose starts
 /datum/reagent/proc/overdose_start(mob/living/M)
+	///SKYRAT EDIT ADDITION: Because these chemicals shouldn't bear the same weight as normal / debatably more harmful chemicals.
+	if(name == "dopamine")///This one also shouldn't have any negative mood effect.
```

Comment:
this is not how you do this in byond

Reply:
also, no, this is an undocumented change in a PR that makes zero mention of it, absolutely not, atomize this

Reply:
> this is not how you do this in byond

it literally worked idk what to tell u